_sass/type-on-strap.scss

```diff
@@ -27,6 +27,7 @@
 @import 'includes/blog_nav';
 @import 'includes/gallery';
 @import 'includes/portfolio';
+@import 'includes/aligner';
```

Comment:
If you are using custom scss for the theme, don't forget to add to get the change. Otherwise the aligner look will be broken.